packages/api/src/endpoints/chat/HelixChatApi.ts

```diff
@@ -251,6 +256,35 @@ export class HelixChatApi extends BaseApi {
 		return new HelixPrivilegedChatSettings(result.data[0]);
 	}
 
+	/**
+	 * Sends a chat message to a broadcaster's chat.
+	 *
+	 * This uses the token of the broadcaster by default.
+	 * If you want to execute this in the context of another user (who has to be moderator of the channel)
+	 * you can do so using [user context overrides](/docs/auth/concepts/context-switching).
+	 *
+	 * @param broadcaster The broadcaster the chat belongs to.
+	 * @param message The message to send.
+	 */
+	async sendChatMessage(
+		broadcaster: UserIdResolvable,
+		message: HelixSendChatMessageParams,
```

Comment:
For simplicity in the common case, as well as for being in line with the `ChatClient` a bit more, I'd prefer if the message text was a separate parameter to the method. The params object should still stay, even though it (currently) only has one property.